be/src/vec/columns/column_nullable.cpp

```diff
@@ -67,6 +67,25 @@ void ColumnNullable::update_hashes_with_value(std::vector<SipHash>& hashes,
     }
 }
 
+void ColumnNullable::update_crcs_with_value(std::vector<uint32_t>& hashes,
+                                            doris::PrimitiveType type,
+                                            const uint8_t* __restrict null_data) const {
+    DCHECK(null_data == nullptr);
+    auto s = hashes.size();
+    DCHECK(s == size());
+    auto* __restrict real_null_data = assert_cast<const ColumnUInt8&>(*null_map).get_data().data();
+    if (doris::simd::count_zero_num(reinterpret_cast<const int8_t*>(real_null_data), s) == s) {
```

Comment:
May be we can unify these two function `this->hash_null()` and `doris::simd::count_zero_num` ? I think it's better to use `!this->hash_null()` here.